### PR TITLE
Update the battery IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-02 20:59:34
Finished: 2018-03-02 21:09:32


<h2>Failing Jobs</h2>
<ul>

<li>chrome:dev</li>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25165)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/348381451)

File: battery-status/battery-interface-idlharness.https.html [hunk 1]
```diff
@@ -7,71 +7,26 @@
...
-};
-[TreatNonObjectAsNull]
-callback EventHandlerNonNull = any (Event event);
-typedef EventHandlerNonNull? EventHandler;
-</script>
-<script type="text/plain">
-partial interface Navigator {
-    Promise<BatteryManager> getBattery ();
-};
+  idl_array.add_untested_idls(dom_idl);
```
Comment (lukebjerring) [inline battery-status/battery-interface-idlharness.https.html]: @Honry - Might be best to use `add_untested_idls(dom_idl, { only: ['Foo', 'Bar'] })` as dom.idl is rather extensive.

Comment (Honry) [reply battery-status/battery-interface-idlharness.https.html]: Yeah, we don't need to test anything from `dom_idl`, so we called `add_untested_idls`, i.e. `dom_idl` will be skipped by `test()`.
`only` is used in `add_tested_idls`,  to only allow specified member to be added to `test()`.

Comment (Honry) [reply battery-status/battery-interface-idlharness.https.html]: Oh, you are right, `only` can also restrict members to be added to `add_untested_idls`.

Comment (Honry) [reply battery-status/battery-interface-idlharness.https.html]: I tried to validate if `only` can be used to restrict members to be added to `add_untested_idls`, seems the answer is no. I set an irrelevant interface to `only`,  no change to test result.

Comment (lukebjerring) [reply battery-status/battery-interface-idlharness.https.html]: I added `only` support for `add_untested_idls` in https://github.com/w3c/web-platform-tests/pull/10100.

When you add irrelevant untested interfaces, the only time that it causes a problem is if `A implements B` and you add that as untested, but `B` is never defined. This is a common problem pulling in large IDL files, as some parts might depend on definitions in other files.

Comment (Honry) [reply battery-status/battery-interface-idlharness.https.html]: Oh, I didn't realize this new change as I used an old branch. And it works now, will update the code soon.

**Review by foolip — APPROVED** [review]
Confirmed the tests to still work. LGTM % updating battery.idl with header and indentation.

**Pull request merged**